Makefile

```diff
@@ -5,29 +5,42 @@ CLEAN := *~
 
 default: build
 
-install.tools: .install.protoc-gen-go .install.grpc-gateway
+install.tools: .install.protoc-gen-go .install.grpc-gateway .install.googleapis
+
+CLEAN += .install.protoc-gen-go .install.grpc-gateway .install.googleapis
 
-CLEAN += .install.protoc-gen-go .install.grpc-gateway
 .install.protoc-gen-go:
-	go get -u -v github.com/golang/protobuf/protoc-gen-go && touch $@
+	cd vendor/github.com/golang/protobuf/protoc-gen-go && go install .
+
+.install.googleapis:
+	mkdir -p vendor/github.com/googleapis && \
+		cd vendor/github.com/googleapis && \
+		git clone https://github.com/googleapis/googleapis
+	touch $@
 
 .install.grpc-gateway:
-	go get -u -v github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger && touch $@
```

Comment:
these can stay in a single command, and the `touch $@` ensures the target's timestamp so that it isn't run every time